frontend/components/events/create/forms/BasicForm.tsx

```diff
@@ -218,7 +221,7 @@ export function BasicInformation({
                 value={price}
                 type="number"
                 onChange={(e) => {
-                  let value = parseInt(e.target.value);
+                  let value = parseFloat(e.target.value);
```

Comment:
we probbaly want to multiply and store it in the frontend as cents straight away